ext/settings.html

```diff
@@ -2028,6 +2028,17 @@ <h1>Yomitan Settings</h1>
                 <button type="button" class="icon-button"><span class="icon-button-inner"><span class="icon" data-icon="material-right-arrow"></span></span></button>
             </div>
         </div></div>
+        <div class="settings-item settings-item-button advanced-only"><div class="settings-item-inner">
+            <div class="settings-item-left">
+                <div class="settings-item-label">Force Anki sync on adding card</div>
+                <div class="settings-item-description">
+                        May cause issues when using in conjuction with Ankiconnect Android, and/or slow or metered connections.
+                    </div>
+            </div>
+            <div class="settings-item-right">
+                <label class="toggle"><input type="checkbox" data-setting="anki.forceSync"><span class="toggle-body"><span class="toggle-track"></span><span class="toggle-knob"></span></span></label>
+            </div>
+        </div></div>
```

Comment:
Breaking up the three bottom buttons in the Anki settings section is unacceptable. How about putting it below `Show card tags and flags`.